package/sbp_protocol/src/sbp_router.yml

```diff
@@ -0,0 +1,112 @@
+
+name: SBP_ROUTER
+ports:
+  - name: SBP_PORT_FIRMWARE
+    pub_addr: "@tcp://127.0.0.1:43010"
+    sub_addr: "@tcp://127.0.0.1:43011"
+    forwarding_rules:
+      - dst_port: SBP_PORT_SETTINGS_DAEMON
+        filters:
```

Comment:
What about something like this, which YAML-only (vs embedding a special DSL in the YAML):

```
filters:
  - action: ACCEPT
    prefix: [0x55, 0xAE, 0x00]
  - action: ACCEPT
    prefix: [0x55, 0xA5, 0x00]
  - action: REJECT
```